scalastyle-config.xml

```diff
@@ -3,7 +3,7 @@
  <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
  <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
   <parameters>
-   <parameter name="maxFileLength"><![CDATA[500]]></parameter>
+   <parameter name="maxFileLength"><![CDATA[525]]></parameter>

```

Comment:
I would vote no, well we just need to split the file, and work on the HA branch.

> On Oct 15, 2015, at 5:12 PM, Noorul Islam K M <notifications@github.com> wrote:
> 
> In scalastyle-config.xml <https://github.com/spark-jobserver/spark-jobserver/pull/284#discussion_r42197287>:
> 
> > @@ -3,7 +3,7 @@
> >   <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
> >   <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
> >    <parameters>
> > -   <parameter name="maxFileLength"><![CDATA[500]]></parameter>
> > +   <parameter name="maxFileLength"><![CDATA[525]]></parameter>
> Can you increase this so that we don't have to keep doing this if we have to some more?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/284/files#r42197287>.
>